products/vmware-harbor-registry.md

```diff
@@ -4,6 +4,10 @@ category: server-app
 tags: cncf
 iconSlug: harbor
 permalink: /vmware-harbor-registry
+alternate_urls:
+-   /harbor-registry
+-   /harbor
```

Comment:
`/harbor` looks good to me, but I don't see the point in adding the other `alternate_urls` : they don't seem commonly used.